configs/securitygroups.yml

```diff
@@ -333,7 +326,12 @@ loaners:
         us-west-2: vpc-cd63f2a4
         us-east-1: vpc-b42100df
     inbound:
-        include: vpn-admin-access
```

Comment:
This was my mistake and not a bug. I left out the leading '-' before the include and so the yaml parsing placed the object directly there instead of creating a single element list.